.drone.star

```diff
@@ -251,7 +264,7 @@ def tests(config):
     'name': 'test',
     'image': 'owncloud/ubuntu:latest',
     'commands': [
-      'curl -sSf http://server:80/',
+      'curl -sSf http://server:80/'
```

Comment:
I thought that nowadays we were putting the trailing comma on the end of the last item in an array. That avoids the need to touch the line again when more commands are added. For example, see line 258.